docker_test.go

```diff
@@ -941,3 +942,32 @@ func TestContainerCreationWithBindAndVolume(t *testing.T) {
 		}
 	}()
 }
+
+func TestContainerWithTmpFs(t *testing.T) {
```

Comment:
what exactly does this test? it looks like it just starts the mysql container then stops it...would it be a good idea to test:
* save something that would be stored in `/var/lib/mysql` in the mysql container
* stop the container
* start another mysql container, ensure that whatever was saved was not persisted (if I understand `tmpfs` correctly)

Reply:
It starts a mysql container with its data volume mapped to a tmpfs volume - that includes a database creation that is part of the bootstrap. I mimicked my particular use case for this and the only observable outcome is that the test is executed faster when using tmpfs.

Probably a better test would be to use a minimal Linux container, no mysql, mount tmpfs volume and assert that data can be written to it. I think Java implementation tests it this way.
I'll try to improve this next week.

Reply:
Updated.
Just like [here](https://github.com/testcontainers/testcontainers-java/blob/bdb82b76a86b57a9789731430cddf3858f41388a/core/src/test/java/org/testcontainers/junit/GenericContainerRuleTest.java#L142-L158), I try to create a file on tmpfs mount.

Reply:
hey @rieske thank you for getting back to me and working on this.

the more I think about it, we are basically just testing the [Docker client Tmpfs functionality](https://godoc.org/github.com/docker/docker/api/types/container#HostConfig) here, so maybe we shouldn't even have a test?  Sorry for going back and forth.

I think this is a good feature.